packages/fxa-auth-server/scripts/clean-up-partial-account-customer.ts

```diff
@@ -88,7 +89,9 @@ const init = async () => {
     random.base32(config.signinUnblock.codeLength) as any // TS type inference is failing pretty hard with this
   );
   // connect to db here so the dry run could get a row count
-  const fxaDb = await db.connect(config, redis);
+  // TBD: Is this needed? The returned DB references is no longer used since we don't need the account manager.
+  //      I don't see where the knex connection is setup. Do we need that somewhere?
+  await db.connect(config, redis);
```

Comment:
How does `AccountCustomers` get its db connection?  (actual question)

Reply:
Ah just followed up on this... AccountCustomers uses knex/objection, and the call to db.connect primes the knex connection, so this is in fact needed. I'll remove the comment.